Frontend/voitheia-App/src/app/components/about/about-us/about-us.component.html

```diff
@@ -1 +1,84 @@
-<p>about-us works!</p>
+<title>W3.CSS Template</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<link rel="stylesheet" href="https://www.w3schools.com/lib/w3-theme-black.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<body id="myPage">
+
```

Comment:
Since we include those html sites inside of our page we do not need all the header and body tags.